code/modules/cm_aliens/structures/special/spawn_pool.dm

```diff
@@ -11,16 +11,18 @@
 	var/surge_incremental_reduction = 3 SECONDS
 	var/mob/melting_body
 
+	var/damage_amount = 20
+
 	luminosity = 3
 
 /obj/effect/alien/resin/special/pool/update_icon()
 	..()
 	overlays.Cut()
 	underlays.Cut()
 	underlays += "[icon_state]_underlay"
-	overlays += image(icon, "[icon_state]_overlay", layer = ABOVE_MOB_LAYER)
+	overlays += mutable_appearance(icon, "[icon_state]_overlay", layer = ABOVE_MOB_LAYER, plane = GAME_PLANE)
 	if(linked_hive.stored_larva)
-		overlays += "[icon_state]_bubbling"
+		overlays += mutable_appearance(icon,"[icon_state]_bubbling", layer = ABOVE_MOB_LAYER + 0.1, plane = GAME_PLANE)
```

Comment:
Are you sure `mutable_appearance` here does anything ? Overlays are immutable once applied

Reply:
Also `GAME_PLANE` should already be the default

Reply:
mutable appearance allows setting the plane, that isnt possible with other icon types